explorer/ast/declaration.h

```diff
@@ -238,8 +240,8 @@ class CallableDeclaration : public Declaration {
         body_(context.Clone(other.body_)),
         virt_override_(other.virt_override_) {}
 
-  void PrintDepth(int depth, llvm::raw_ostream& out) const;
-
+  void PrintIndent(int indent_num_spaces,
+                   llvm::raw_ostream& out) const override;
```

Comment:
So you know, I'm looking at this override and I think we should probably remove it (inlining the code) to reduce complexity and be more consistent (i.e., removing the indent special-casing for callables). It'd also mean PrintIndent wouldn't need to be virtual. I'm fine with it for now since the function was there before, I just want you to know the line of thought.

Reply:
Thanks for letting me know about it, removing the override and making the `PrintIndent` non virtual in the next commit.